services/api/src/resources/env-variables/resolvers.ts

```diff
@@ -672,44 +701,57 @@ export const deleteEnvVariable: ResolverFn = async (
   });
 
   let resource: AuditResource;
+  let orgId;
   const rows = await query(sqlClientPool, Sql.selectEnvVarById(id));
   const envVar = R.prop(0, rows);
   if (envVar.organization) {
     const org =
       await orgHelpers(sqlClientPool).getOrganizationById(envVar.organization);
     resource = {
-      id: org.id,
+      id: org.id.toString(),
       type: AuditType.ORGANIZATION,
       details: org.name,
     };
+    orgId = org.id;
   } else if (envVar.project) {
     const project =
       await projectHelpers(sqlClientPool).getProjectById(envVar.project);
     resource = {
-      id: project.id,
+      id: project.id.toString(),
       type: AuditType.PROJECT,
       details: project.name,
     };
+    if (project.organization) {
+      orgId = project.organization
+    }
   } else if (envVar.environment) {
     const environment =
       await environmentHelpers(sqlClientPool).getEnvironmentById(envVar.environment);
+    const project =
+      await projectHelpers(sqlClientPool).getProjectById(environment.id);
```

Comment:
is `environment.id` correct?
```suggestion
      await projectHelpers(sqlClientPool).getProjectById(environment.project);
```